drivers/gles2/rasterizer_canvas_gles2.cpp

```diff
@@ -2020,8 +2020,8 @@ void RasterizerCanvasGLES2::render_joined_item(const BItemJoined &p_bij, RenderI
 			}
 
 			if (shader_ptr != r_ris.shader_cache) {
-				if (shader_ptr->canvas_item.uses_time) {
-					VisualServerRaster::redraw_request();
+				if (shader_ptr->canvas_item.uses_time && OS::get_singleton()->_are_low_priority_redraws_enabled()) {
+					VisualServerRaster::redraw_request(false);
```

Comment:
Ah the check above is redundant as the request uses false, I'll fix that in the morning.